examples/http_ex/README.md

```diff
@@ -0,0 +1,9 @@
+# HTTP agent example
```

Comment:
Please move to `docs/http_connection_and_skill.md`

Reply:
moved to `docs/http-connection-and-skill.md`